templates/user/talk/talk_edit.html.twig

```diff
@@ -0,0 +1,35 @@
+{% extends "user/user_layout.html.twig" %}
+
+{% form_theme form 'bootstrap_4_layout.html.twig' %}
+{% form_theme form.intro _self %}
+
+{% block nav %}
+    {{ parent() }}
+    <a href="{{ path('user_talks') }}" class="ml-auto">
+        <div class="btn btn-primary">Back to talks</div>
+    </a>
+{% endblock %}
+
+{% block _edit_talk_intro_widget %}
+    <textarea class="form-control" name="{{ full_name }}" rows="10">{{ value }}</textarea>
+{% endblock %}
+
+{% block content %}
+    <div class="row flex-column align-items-center w-100">
+        <h2>Edit your talk</h2>
+        <div class="row flex-column align-items-center w-50">
+            {{ form_start(form, {
+                attr: {
+                    class: 'row flex-column w-100',
+                    id: form.vars.id
```

Comment:
why this line?

Reply:
If I don't add this, the form doesn't have an ID and I need it to have one beacause I use it in the tests

Reply:
That's weird 🤔  but ok for now